Clean and simplify POI panel structure

bbecquet opened the pull request:
## Description
Simplify stuff about the POI Panel, trying to reduce its complexity:
- regroup component files in the same directory
- use the `<Panel>` component, when it's not in compact mode (aka. "card")
- remove a lot of useless/specific CSS
- slightly improve some UX details in the process, like the close button on the mobile card which is consistently present and never collides with the action buttons anymore (fixing a part of QMAPS-1066)

## Why
- Improve style cohesion everywhere: now all panels use the standard `<Panel>` component, ensuring visual changes to it are applied everywhere
- Simplify code readability and maintenability

amatissart commented:
On Chrome mobile the Poi panel is not correctly positioned
<img src="https://user-images.githubusercontent.com/4726554/72145942-cd90fc80-339b-11ea-9677-06a4a7f46f0d.png" width="400px"/>

bbecquet commented:
@amatissart :wave: 

> On Chrome mobile the Poi panel is not correctly positioned
> <img alt="" width="400px" src="https://user-images.githubusercontent.com/4726554/72145942-cd90fc80-339b-11ea-9677-06a4a7f46f0d.png">

I think I fixed it. In any case I can't reproduce it on my device, but I'm interested in you testing in again :)
I rebased this PR on [the one](https://github.com/QwantResearch/erdapfel/pull/516) including panel DOM simplifications, so I'll let the **WIP** flag until the other is merged.

bbecquet commented:
Rebase on master done, removing WIP and ready to review :)

amatissart commented on src/panel/poi/PoiPanel.jsx:
```diff
@@ -202,94 +192,57 @@ export default class PoiPanel extends React.Component {
+      }
+      {!backAction && pagesjaunes &&
+        <div className="poi_panel__pj_logo_container">{pagesjaunes}</div>
+      }
```
This block should be rendered when accessing "pagesjaunes" POIs via their URL, but `backAction` is always defined...

bbecquet replied:
Thanks for your vigilance :) :policeman: 
I think I fixed this behavior.
